client/consensus/manual-seal/src/lib.rs

```diff
@@ -49,6 +49,11 @@ pub use self::{
 };
 use sp_api::{ProvideRuntimeApi, TransactionFor};
 
+/// The `ConsensusEngineId` of Manual Seal.
+// We should consider creating a new crate primitives/manual-seal for this, if it ends up being used
+// outside of this crate.
+pub const MANUAL_SEAL_ENGINE_ID: ConsensusEngineId = [b'm', b'a', b's', b'e'];
```

Comment:
`manl`?

Reply:
MAnual SEal ... yeah I know not the best ... :(

Reply:
I'll change it ..